src/renderer/coremods/settings/index.tsx

```diff
@@ -12,13 +12,15 @@ export default class SettingsMod extends Coremod {
 
   start() {
     settingsTools.addAfter("Billing", [
+      /* eslint-disable */
```

Comment:
Why?

Reply:
Because the linter hates when the function name is capitalized but it's not used as a class. Hopefully temporary, but this way eslint doesn't bother me while I'm focusing on more important stuff.